presto-docs/src/main/sphinx/release/release-0.188.rst

```diff
@@ -0,0 +1,43 @@
+=============
+Release 0.188
+=============
+
+General Changes
+---------------
+
+* Fix exception for illegal inputs to array :func:`slice` function.
+* Fix inverted sign for time zones ``Etc/GMT-12``, ``Etc/GMT-11``, ..., ``Etc/GMT-1``,
+  ``Etc/GMT+1``, ... ``Etc/GMT+12``.
+* Add environment support to db-backed resource groups manager.
+  Previously, configurations of different clusters have to be stored in separate databases.
+  With this change different cluster configurations can be stored in the same table and
+  Presto will use the new `environment` column to differentiate them.
+  The `node.environment` config property will be used to determine the current environment,
+  and Presto will load the configuration for that environment on startup.
+* Add `pages-index.eager-compaction-enabled` config property.
+  For memory-intensive JOINs, currently, we do lazy compaction to reduce memory usage.
+  This leads to concentrated rapid allocation, causing excessive GC pauses.
+  Enabling this property may increase CPU time for all JOINs.
+* Improve performance of server logging and HTTP request logging.
+* Improve CPU performance and reduce GC overhead for compaction of in-memory data structures,
+  primarily used in JOINs.
+* Mitigate excessive GC and degraded query performance by forcing expiration of
+  generated classes for function implementation and SQL expressions an hour after its generation.
+
+CLI Changes
+-----------
+
+* Fix transaction support. Previously, the transaction was marked as cleared
```

Comment:
Fix transaction support. Previously, after the first statement in the
transaction, the transaction would be abandoned and the session would
silently revert to auto-commit mode.